go/tasks/plugins/hive/execution_state.go

```diff
@@ -344,11 +345,13 @@ func KickOffQuery(ctx context.Context, tCtx core.TaskExecutionContext, currentSt
 
 	taskExecutionIdentifier := tCtx.TaskExecutionMetadata().GetTaskExecutionID().GetID()
 	commandMetadata := client.CommandMetadata{TaskName: taskName,
-		Domain:        taskExecutionIdentifier.GetTaskId().GetDomain(),
-		Project:       taskExecutionIdentifier.GetNodeExecutionId().GetExecutionId().GetProject(),
-		Labels:        tCtx.TaskExecutionMetadata().GetLabels(),
-		AttemptNumber: taskExecutionIdentifier.GetRetryAttempt(),
-		MaxAttempts:   tCtx.TaskExecutionMetadata().GetMaxAttempts(),
+		Domain:              taskExecutionIdentifier.GetTaskId().GetDomain(),
+		Project:             taskExecutionIdentifier.GetNodeExecutionId().GetExecutionId().GetProject(),
+		Labels:              tCtx.TaskExecutionMetadata().GetLabels(),
+		AttemptNumber:       taskExecutionIdentifier.GetRetryAttempt(),
+		MaxAttempts:         tCtx.TaskExecutionMetadata().GetMaxAttempts(),
+		WorkflowExecutionID: taskExecutionIdentifier.NodeExecutionId.ExecutionId.Name,
```

Comment:
nit (for consistency):
```suggestion
		WorkflowExecutionID: taskExecutionIdentifier.GetNodeExecutionId().GetExecutionId().GetName(),
```

Reply:
addressed